content/installation/java/Java-linux-packages.md

```diff
@@ -0,0 +1,157 @@
+
+#### Logging
+
+The Exec Helper logs messages to the host's Syslog service using the identifier "Contrast". The Exec Helper uses the "user.warn" and "user.debug" Syslog facility and level, respectively. Use `journalctl` to view the messages (i.e., `journalctl -t Contrast`). Traditional init.v systems, including Ubuntu 14.04, store Syslog messages in the file */var/log/messages* by default instead of the systemd Journal.
+
+
+## Red Hat-Based Systems
+
+### Install the Java agent
+
+Complete the following steps to install the Contrast Java agent for Red Hat Enterprise Linux (RHEL) and CentOS versions 5, 6 and 7.
+
+* To install Java agent from Contrast's Yum repository, configure your system to use the repository.
+
+```
+OSREL=$(rpmquery -E "%{rhel}")
+sudo -E tee /etc/yum.repos.d/contrast.repo << EOF
+[contrast]
+name=contrast repo
+baseurl=https://contrastsecurity.jfrog.io/contrastsecurity/rpm-staging/centos-$OSREL/
+gpgcheck=0
+enabled=1
+EOF
+```
+
+* Once you've finished configuration, install the Contrast Java agent.
+
+```
+yum install contrast-java-agent
+```
+
+* Edit the */etc/contrast/contrast-java-config.yaml* file to configure Contrast Java agent to connect to the Contrast application. <!-- (Go to the YAML [Configuration doc](_________) for more info.) -->
+
+### Next steps
+
+At this stage, you may either complete the agent installation process by follow the instructions for the [Java application container](installation-javainstall.html) of your choice, or continue on to **Add the Exec Helper** package.
+
+### Add the Exec Helper
+
+Complete the following steps to install the Exec Helper package.
+
+* Before you install the package, use the following command to verify that you have already installed the Contrast Java agent.
+
+```
+yum list installed contrast-java-agent
+```
+
+* If it isn't installed, follow the instructions in the **Red Hat-Based Systems** section to install and configure the `contrast-java-agent` package.
+
+* As you already configured your system to use the Contrast Yum repository when you installed the `contrast-java-agent` package, you can install the `contrast-java-agent-exec-helper` package.
+
+```
+yum install contrast-java-agent-exec-helper
+```
+
+* Verify that the Exec Helper is working by executing `java` and confirming that Contrast starts by observing console messages.
+
+```
+$ bash -c "java -Dcontrast.stdout=true -version 2>1 | grep Contrast | head -n 1"
+[Contrast] Wed Aug 15 17:37:23 UTC 2018 No TeamServer configuration detected. Agent will only be reporting to local listeners (e.g., Eclipse Plugin).
+```
+
+* The Contrast Java agent Exec Helper affects all new shells, as indicated by the `bash -c` in the verification command. Consequently, it will be present in your current shell. Reload your shell to enable the Contrast Java agent Exec Helper. Restart any `java` services to enable Contrast for those services.
```

Comment:
Same again:

Should this read

> it will **not** be present in your current shell

Which is why you should reload your shell to enable it?